typos.toml

```diff
@@ -0,0 +1,5 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
```

Comment:
I think it's better to use a hidden type file, like `.typos.toml`.

Reply:
Done.